chiselFrontend/src/main/scala/chisel3/core/Bits.scala

```diff
@@ -1011,3 +1012,33 @@ object FixedPoint {
   }
 
 }
+
+/** Data type for Verilog inout */
+final class Analog private (width: Width) extends Element(width) {
+  require(width.known, "Since Analog is only for use in BlackBoxes, width must be known")
+
+  // Used to enforce single bulk connect of Analog types, multi-attach is still okay
```

Comment:
To prevent unintuitive behavior surrounding last connect semantics. Basically:
```scala
mod1.someBundle <> mod2.someBundle
mod1.someBundle <> mod3.someBundle
```
Last connect semantics say that the latter will overrule the former, but in the case of any Analog members, they would stay connected. Firrtl doesn't even allow attach in connections but Analog would be virtually unusable in Chisel without it so the most reasonable contract I could come up with was that you *can* bulk connect, but not more than once because last connect doesn't really make sense